images/disks-images-provider/entrypoint.sh

```diff
@@ -27,9 +27,13 @@ mkdir -p /images/datavolume1 /images/datavolume2 /images/datavolume3
 echo "converting cirros image from qcow2 to raw, and copying it to local-storage directory, and creating a loopback device from it"
 # /local-storage will be mapped to the host dir, which will also be used by the local storage provider
 qemu-img convert -f qcow2 -O raw /images/cirros/disk.img /local-storage/cirros.img.raw
-LOOP_DEVICE=$(losetup --find --show /local-storage/cirros.img.raw)
+
+mknod /dev/loop99 b 7 0
```

Comment:
Would `mknod` fail if the container restarts?

Reply:
Ah that is probably not an issue, since that would mean that the container is set up again and `/dev/loop99` does not exist then.

Reply:
To be honest I did not think about that possibility but yes, it should not. I also made a quick try that confirms that. If I stop & restart a container `/dev/loop99` is gone